### PR TITLE
Fix arg1 of dt.datetime.combine ()

### PR DESCRIPTION
**Pull request by pdav**
call to: ``dt.datetime.combine (dt.date, dt.time(dt.datetime.now().hour))``
fails with:
``TypeError: combine() argument 1 must be datetime.date, not type``

Test with: start ikhal, then press the ``n`` key to add a new event.

### CODE REVIEW

**Comment by geier** [comment]
Ouch, I messed that one up during the datetime import refactor last week.

File: khal/ui/__init__.py [hunk 1]
```diff
@@ -870,7 +869,7 @@ def new(self, date, end=None):
             self.pane.window.alert(('alert', 'No writable calendar.'))
             return
         if end is None:
-            start = dt.datetime.combine(dt.date, dt.time(dt.datetime.now().hour))
+            start = dt.datetime.combine(dt.date.today (), dt.time(dt.datetime.now().hour))
```
Comment (geier) [inline khal/ui/__init__.py]: This should actually be `dt.datetime.combine(date, dt.time(dt.datetime.now().hour())`, date is a variable given to `new()` here.

**Comment by pdav** [comment]
Whops! New commit added.

**Comment by geier** [comment]
thank you!

**Pull request merged**